unison-src/transcripts/merge.output.md

```diff
@@ -579,9 +624,21 @@ baz = "bobs baz"
 ```ucm
 project/alice> merge /bob
 
-  I couldn't automatically merge project/bob into project/alice.
+  I couldn't automatically merge project/bob into alice.
```

Comment:
Was it intentional that the target project name went away? I noticed it wasn't in one of the messages PRs, but didn't notice it had been in the earlier version of the message, and I just assumed it was hard to produce for some reason.

Reply:
It was - I thought that because you are always merging into the current project branch, showing the project is a little wordy.

But it especially looks weird alongside a project+branch for Bob, so I can add it back. Or, I can make it so the project is omitted from Bob if it matches Alice's, so that the very common case is less verbose overall.

Reply:
I reverted this for now